### PR TITLE
Adds warning that the ES connector is unstable.

### PR DESCRIPTION
**Pull request by susodapop**
## Type of PR
- [x] Change

## Description

Adds a note to the ElasticSearch document that the connector is buggy.

## Related Documents & Tickets

None

## Screenshots

(see callout at top of page)

Screen Shot 2020-08-17 at 12 57 58

### CODE REVIEW

File: src/pages/kb/data-sources/elasticsearch.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ parent_category: data-sources
 title: Elasticsearch
 slug: elasticsearch
 ---
+{% callout danger %}
+
+The Redash ES connector was a community contribution from several years ago. There are many bugs and it needs to be rewritten. Work on this is currently underway (see [here](https://github.com/getredash/redash/pull/4391)) but its release date is TBD.
```
Comment (arikfr) [inline src/pages/kb/data-sources/elasticsearch.md:9]: Let's reword this to say: work was started, but currently blocked as we don't have the resources to test the new implementation. We're happy for assistance if someone wants to help.

Comment (susodapop) [reply src/pages/kb/data-sources/elasticsearch.md]: Done!

**Review by arikfr — APPROVED** [review]
